### ISSUE TITLE
Compatability Of `protoc-gen-gogofaster`  When Use For gRPC With Non Go RPC Clients?

### ISSUE THREAD

**Issue by postables** [gogo/protobuf#609]
Hello, I cant seem to find any information on whether or not gRPC server code generated with `protoc-gen-gogofaster` is compatible with gRPC clients that aren't using the Golang programming language?

ie, I use `protoc-gen-gogofaster` for a golang server application I'm writing, and expect clients to be built using languages like Java, JavaScript, or Python. Will clients written in other languages work with gRPC server code generated using `protoc-gen-gogofaster` ?

**Comment by tegk** [comment]
Yes, Gogo is just an implementation of gRPC and the clients can use their implementation they want to use without a problem.
